Makefile

```diff
@@ -367,10 +371,10 @@ docker-buildx-crds-release: build-crds docker-buildx-builder
 
 # Build gator image
 docker-buildx-gator-dev: docker-buildx-builder
-	docker buildx build --build-arg LDFLAGS=${LDFLAGS} --platform "linux/amd64,linux/arm64,linux/arm/v6"\
```

Comment:
Were these platform versions removed for local testing? If so should we add this back?

Reply:
Good catch!! I missed this